tests/test_weighted_pandas.py

```diff
@@ -188,6 +189,16 @@ def test_WeightedSeries_quantile():
         assert_allclose(quantile, q, atol=1e-2)
 
 
+def test_WeightedSeries_hist():
+    series = test_WeightedSeries_constructor()
+    ax = series.hist(bins=20, density=True)
+    assert len(ax.patches) == 20
+    norm = 0
+    for patch in ax.patches:
+        norm += patch.get_height() * patch.get_width()
+    assert norm == pytest.approx(1)
```

Comment:
Any idea why this is failing but the equivalent one for `WeightedDataFrame` is not?

The following runs locally without a problem for me:
```python
import pytest
from anesthetic.weighted_pandas import WeightedSeries

series = WeightedSeries(np.random.randn(1000), weights=np.random.rand(1000))

ax = df.hist(bins=20, density=True)

assert len(ax.patches) == 20

norm = 0
for patch in ax.patches:
    norm += patch.get_height() * patch.get_width()
print(norm)
assert norm == pytest.approx(1)
```

Reply:
Now that was a non-trivial debug. Was able to replicate it locally, but only if running the tests in series with:
```bash
python -m pytest -k hist
```
Running it in parallel as I usually do with
```bash
./bin/run_tests
```
passed. I suspect this is therefore to do with running the suite in such a way that means that, when calling the parent function `Series.hist` or `WeightedFrame.hist` whilst other axes are still open, it tries to replot on a still open axis (as it would do if you call `plt.plot(...)` when there is  an axis that has not yet been closed).

Explicitly closing the plots seems to have fixed it.

Reply:
Nice! I had the same issue, running in parallel worked but in series didn't. I tried around with `plt.close` but didn't know about the `"all"` option.